### PR TITLE
Add header in index.html + styles, scripts, images for website

### PR DESCRIPTION
**Pull request by in-siders17**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,144 @@
...
+		<meta name="author" content="Daniel Tkachenko">
+		<meta name="facebook-domain-verification" content="d6epnlp2evd0qt2uzwuot4jegcw1jl">
+		<link rel="icon" href="https://www.tattoo-lamarch.studio/tattoo.ico">
+
+
+		<title>TATTOO LAMARCH STUDIO</title>
+
+		<!-- fonts -->
+		<link href="styles/css" rel="stylesheet">
+		<link href="styles/css(1)" rel="stylesheet">
```
Comment (dindin28) [inline index.html:17]: Can it be merged in single css?

File: index.html [hunk 2]
```diff
@@ -1,10 +1,144 @@
...
+          gtag('js', new Date());
+
+          gtag('config', 'AW-11014528428');
+        </script>
+	<meta http-equiv="origin-trial" content="A727AcAeLCei/ZogJHBlJUS63SxP6AeIROo7qXrkjrxkoxBu0eSSmypAHmGYwk4HjBMQp5nxCFODzfVnUIe31AQAAAB7eyJvcmlnaW4iOiJodHRwczovL3d3dy5nb29nbGV0YWdtYW5hZ2VyLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjg4MDgzMTk5LCJpc1RoaXJkUGFydHkiOnRydWV9"><script type="text/javascript" async="" src="scripts/f.txt"></script><meta http-equiv="origin-trial" content="A727AcAeLCei/ZogJHBlJUS63SxP6AeIROo7qXrkjrxkoxBu0eSSmypAHmGYwk4HjBMQp5nxCFODzfVnUIe31AQAAAB7eyJvcmlnaW4iOiJodHRwczovL3d3dy5nb29nbGV0YWdtYW5hZ2VyLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjg4MDgzMTk5LCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A727AcAeLCei/ZogJHBlJUS63SxP6AeIROo7qXrkjrxkoxBu0eSSmypAHmGYwk4HjBMQp5nxCFODzfVnUIe31AQAAAB7eyJvcmlnaW4iOiJodHRwczovL3d3dy5nb29nbGV0YWdtYW5hZ2VyLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjg4MDgzMTk5LCJpc1RoaXJkUGFydHkiOnRydWV9"><script type="text/javascript" async="" src="scripts/f(1).txt"></script></head>
```
Comment (dindin28) [inline index.html:44]: check if "f(1).txt" can be renamed to "f(1).txt"

File: index.html [hunk 3]
```diff
@@ -1,10 +1,144 @@
...
+								</a>
+							</div>
+						</div>
+						<div class="col-10">
+							<ul class="nav-ul header">
+								<div class="nav-ul header exit">
+									<span></span>
+								</div>
+								<li class="nav-ul-li">
+									<a href="https://www.tattoo-lamarch.studio/#sec1" class="nav-ul-li-a">Про нас</a>
```
Comment (dindin28) [inline index.html:78]: Make relative page "/#about-us"

Check what better naming for html classes
"about-us" or "about_us"

File: index.html [hunk 4]
```diff
@@ -1,10 +1,144 @@
...
+								<li class="nav-ul-li">
+									<a href="https://www.tattoo-lamarch.studio/#sec2" class="nav-ul-li-a">Послуги</a>
+								</li>
+
+								<li class="nav-ul-li">
+									<a href="https://www.tattoo-lamarch.studio/#sec3" class="nav-ul-li-a">Відгуки</a>
+								</li>
+
+								<li class="nav-ul-li">
+									<a href="https://www.tattoo-lamarch.studio/#sec4" class="nav-ul-li-a">Команда</a>
```
Comment (dindin28) [inline index.html:90]: Make relative page "/#our-team"

File: index.html [hunk 5]
```diff
@@ -1,10 +1,144 @@
...
+								</li>
+
+								<li class="nav-ul-li">
+									<a href="https://www.tattoo-lamarch.studio/#sec5" class="nav-ul-li-a">Галерея</a>
+								</li>
 
+								<li class="nav-ul-li">
+									<a href="https://www.tattoo-lamarch.studio/#sec6" class="nav-ul-li-a">Контакти</a>
```
Comment (dindin28) [inline index.html:98]: Make relative page "/#contacts"

File: index.html [hunk 6]
```diff
@@ -1,10 +1,144 @@
...
+				</div>
+			</nav>
+			<section>
+				<div class="container-fluid">
+					<div class="row">
+						<div class="col-lg-1"></div>
+						<div class="col-lg-10">
+							<div class="h-title">
+								<div class="h-title-t1 wow pulse" style="visibility: visible; animation-name: pulse;">
+									TATTOO LAMARCH STUDIO
```
Comment (dindin28) [inline index.html:117]: FELISSS TATTOO

File: index.html [hunk 7]
```diff
@@ -1,10 +1,144 @@
...
+										Записатись
+									</a>
+								</div>
+							</div>
+						</div>
+					</div>
+					<div class="row">
+						<div class="col-md-12">
+							<div class="h-bottom wow pulse" data-wow-delay="0.5s" style="visibility: visible; animation-delay: 0.5s; animation-name: pulse;">
+								<a href="tel:+380940000003"><span>Тел.</span> +38 (094) 000-000-3</a>
```
Comment (dindin28) [inline index.html:135]: tel:+380973074604 
+38 (094) 000-000-3 -> 097 307 46 04

File: index.html [hunk 8]
```diff
@@ -1,10 +1,144 @@
...
+								<div><span>Рабочі години:</span> 10:00 - 22:00</div>
```
Comment (dindin28) [inline index.html:136]: 8:00 - 18:00

**Review by dindin28 — CHANGES_REQUESTED** [review]

File: index.html [hunk 9]
```diff
@@ -1,10 +1,144 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="uk"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
 <head>
-    <meta charset="UTF-8">
-    <title>Title</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+		<meta name="description" content="Самый молодёжный тату салон в городе. LUXURY комфорт и атмосфера.
```
[the remark this answers is not in the document]
Comment (in-siders17) [reply index.html:5]: Complete

File: index.html [hunk 10]
```diff
@@ -1,10 +1,144 @@
...
+		Исключительно новый уровень тату индустрии в Виннице.">
+		<meta name="keywords" content="tattoo lamarch studio, тату винница, винница тату, ламарч, тату ламарч винница, тату ламарч">
```
[the remark this answers is not in the document]
Comment (in-siders17) [reply index.html:7]: Complete

File: index.html [hunk 11]
```diff
@@ -1,10 +1,144 @@
...
+		<meta name="author" content="Daniel Tkachenko">
+		<meta name="facebook-domain-verification" content="d6epnlp2evd0qt2uzwuot4jegcw1jl">
+		<link rel="icon" href="https://www.tattoo-lamarch.studio/tattoo.ico">
```
[the remark this answers is not in the document]
Comment (in-siders17) [reply index.html:10]: edited complete

File: index.html [hunk 12]
```diff
@@ -1,10 +1,144 @@
...
+
+
+		<title>TATTOO LAMARCH STUDIO</title>
```
[the remark this answers is not in the document]
Comment (in-siders17) [reply index.html:13]: edited compliete

**Pull request merged**